packages/terra-responsive-element/src/terra-dev-site/doc/responsive-element/ResponsiveElement.1.doc.mdx

```diff
@@ -27,12 +36,19 @@ This component requires the following peer dependencies be installed in your app
 | react | ^16.8.5 |
 | react-dom | ^16.8.5 |
 
-
-
 <!-- AUTO-GENERATED-CONTENT:END -->
 
```

Comment:
Can you add the "Usage" section

Reply:
updated here https://github.com/cerner/terra-core/pull/2864/commits/f5f7237a3c1a82f674819126e1c3a761bbc48165.